src/account-info/AccountDetails.js

```diff
@@ -19,6 +20,15 @@ type Props = {
   presence: Presence,
 };
 
+const screenWidth = Dimensions.get('window').width;
```

Comment:
You should consider the reworking of `screenWidth` to be a separate commit, as it is not directly related to the functionality you are implementing.

Also, extracting it like that has both a positive and a negative:
* Positive: you are reusing the object so no recreating of the object on each render. That is good, I hope that was your intention.
* Negative: this is initialized once and reused in each subsequent renders. That is not desirable as the dimensions of the window can actually change. For example if you rotate the phone, or with the new functionality both in Android and iOS for multitasking where you split or resize the screen.

I would revert that change, and consider a different approach in a separate PR.
You can probably achieve a 'stretch the image' functionality without using `Dimensions.get`.